api/v0/serializers.py

```diff
@@ -1,23 +1,84 @@
 import json
 
+# FIXME: serializers.ValidationError not exceptions.ValidationError
```

Comment:
I meant only for my new methods in the serialiser. I've updated the other occurrences there, tests still pass.